README.md

```diff
@@ -16,12 +16,48 @@ Communication
 The easiest way to [communicate with us is by opening a new issue](https://github.com/savoirfairelinux/ringme.js/issues/new).
 Make sure you add proper labels to your issue (e.g. question, issue, enhancement, …)
 
+Usage
+-----
+
+### Why you don't need it
+
+This project does nothing except proposing a default look & feel for a
+`Ring Me` button that you can add to any web page _easily_.
```

Comment:
This project does nothing other than to propose a default look and feel for a `Ring Me` button that you can _easily_ add to any Web page.

Reply:
Fixed by 40e83fb